lib/PuppeteerSharp/ContextPayload.cs

```diff
@@ -1,13 +1,20 @@
-﻿using Newtonsoft.Json;
-
 namespace PuppeteerSharp
 {
     internal class ContextPayload
     {
+        /// <summary>
+        /// Gets or sets the Id.
+        /// </summary>
         public int Id { get; set; }
 
+        /// <summary>
+        /// Gets or sets the context payload aux data.
+        /// </summary>
         public ContextPayloadAuxData AuxData { get; set; }
 
+        /// <summary>
+        /// Gets or sets the name.
+        /// </summary>
         public string Name { get; set; }
```

Comment:
We are not adding XML Docs for internal classes.

Reply:
Thanks for the review @kblok.